### PR TITLE
Src 5502 changing right ethercat variable

### PR DESCRIPTION
**Pull request by georgiablanco**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -294,6 +294,12 @@ Before executing any tests, it is very useful to make sure you have unlimited sc
 
 Once you have written your code for aurora in your branch, test it locally with Molecule first before pushing to GitHub.
 
+There are some molecule_docker tests which require connecting to AWS to download files (such as downloading the hand manual). For this reason, before running any Molecule tests, ask the system administrator for your AWS access key and secret access key. Then, in the docker container terminal, type:
```
Comment (biofotis) [inline README.md:297]: Please use separate PRs next time. This is not related to changing the variable name

**Review by ToivoS — APPROVED** [review]

**Review by mikramarc — APPROVED** [review]
